### DISCUSSION TITLE
Should we incorporate `ObservableMacro` macro from `swift-syntax` examples?

### DISCUSSION THREAD

**Discussion by Matejkob** [pointfreeco/swift-macro-testing#9] [General]
Hey there!

While diving deep into our unit tests, I observed that we've adeptly incorporated most of the macro examples from the `swift-syntax` repository. However, one particular macro seems to have slipped through the cracks: the `ObservableMacro`. I'm genuinely curious about this. Were there specific reasons or considerations behind leaving it out? Let's discuss and potentially evaluate its integration if it aligns with our library's goals.

**Comment by mbrandonw** [comment]
Hi @Matejkob, I think that is just an oversight on our part, so if you have time to PR adding it in we'd happily accept it!

**Reply by Matejkob** [reply]
Sure think! I've added it in the [`Incorporate ObservableMacro`](https://github.com/pointfreeco/swift-macro-testing/pull/10/commits/db03afc2222a1f7acaca1d81bdb0125c30e619ee) commit.
